### PR TITLE
viclim/improve maintainability

### PR DESCRIPTION
**Pull request by viclim**
Upgraded configurations in `.rubocop.yml` to use `rubocop@0.52.1`.

### CODE REVIEW

File: app/classes/document_query.rb [hunk 1]
```diff
@@ -115,6 +115,7 @@ def suggestion_highlight
     }
   end
 
+  # rubocop:disable Metrics/CyclomaticComplexity, Metrics/MethodLength
```
Comment (viclim) [inline app/classes/document_query.rb]: Disable rubocop check for this function as this block of code is using the elasticsearch DSL. Splitting it up will makes the code harder to read IMO.

**Comment by viclim** [comment]
Just realize https://github.com/GSA/i14y/pull/28 is doing the something about `rubocop`.

**Comment by viclim** [comment]
@nickmarden This PR is ready to be reviewed.

**Comment by nickmarden** [comment]
@MothOnMars could you take a look? This looks right to me but I don't just want to :shipit: without a second set of eyeballs on it.

**Comment by MothOnMars** [comment]
Thanks, @viclim . I didn't realize that I had never merged https://github.com/GSA/i14y/pull/28. I'm going to work on merging that, which includes additional cops added in the updated rubocop version. When that's done, then this can be rebased. Overall, the changes look good.

**Comment by MothOnMars** [comment]
@viclim , https://github.com/GSA/i14y/pull/28 has been merged; please rebase.

**Comment by nickmarden** [comment]
@viclim could you take a look at the two CodeClimate issues?

**Pull request closed without merging**